core/src/test/java/org/commonjava/maven/ext/manip/io/ModelResolverTest.java

```diff
@@ -52,9 +51,7 @@ public void resolveArtifactTest()
                             "cache-dir" ) );
         final GalleyAPIWrapper wrapper = new GalleyAPIWrapper( galleyInfra );
         final ModelIO model = new ModelIO();
-        final Field f = ModelIO.class.getDeclaredField( "galleyWrapper" );
-        f.setAccessible( true );
-        f.set(model, wrapper);
+        FieldUtils.writeField( model, "galleyWrapper", wrapper, true );
 
```

Comment:
Again, why not create a constructor in ModelIO that allows it to specify the minimal components it needs? I see how FieldUtils is nicer than straight reflection, but it's still reflection underneath it all...